### DISCUSSION TITLE
Why is the .deb and .rpm not auto-updateable like AppImage and Snap ?

### DISCUSSION THREAD

**Discussion by IkelAtomig** [tom-james-watson/breaktimer-app#275] [General]
I think it would be easier to add them to the respective package managers ?

I am asking because mostly people seriously hate Snaps in Linux Community. Flatpak isn't available for the app yet due to Electron Builder. If it was ready. This would be totally unnecessary.

**Comment by tom-james-watson** [comment]
Because getting them into distros' package managers is a huge undertaking and committment that I don't want to undertake, basically.

**Reply by IkelAtomig** [reply]
I understand. Then how about a PPA for Debian and Copr for RPM then, you do it as you wish.

**Reply by tom-james-watson** [reply]
That's also a lot of work and a lot of new maintenance. I'm happy with the current options, honestly.

**Reply by IkelAtomig** [reply]
Okay, Understandable. But did you try building the app in flatpak though despite the documentation by electron-builder or you could port the app to Tauri too.

**Reply by tom-james-watson** [reply]
If someone wants to provide a pr that does all that then maybe, but I'm not planning on doing any of that any time soon myself.
